passkey-types/src/webauthn/attestation.rs

```diff
@@ -569,13 +791,78 @@ pub enum ClientDataType {
     /// Serializes to the string `"webauthn.get"`
     #[serde(rename = "webauthn.get")]
     Get,
+
+    /// Serializes to the string `"payment.get"`
+    #[serde(rename = "payment.get")]
+    PaymentGet,
+
+    /// Serialize to an empty string `""`
+    #[serde(rename = "")]
+    Empty,
+}
+
+impl fmt::Display for ClientDataType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let renamed = match self {
+            ClientDataType::Create => "webauthn.create",
+            ClientDataType::Get => "webauthn.get",
+            ClientDataType::PaymentGet => "payment.get",
+            ClientDataType::Empty => "",
+        };
+        write!(f, "{}", renamed)
```

Comment:
**Suggestion:** Instead of keeping this in sync with the serde attributes. I would suggest using it here:

```suggestion
        // SAFETY: if this unwinds it is developer error
        let renamed = serde_json::to_string(self).unwrap();
        write!(f, "{}", renamed.trim_matches('"'))
```

It requires an allocation unfortunately but it allows less duplication. Also It seems to only be useful in tests here. If its used extensively outside of this codebase I'm willing to revisit this.

Reply:
Sure makes sense. Thanks for the detailed suggestion!